.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Avocado Gating CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Logical, stylistic, analytical and formatter checks
+    runs-on: ubuntu-latest
```

Comment:
Although there is no rpm-based distro, happy to see Windows and macOS, we could improve our tests on those platforms, although I know we do not officially support them.

Reply:
> Although there is no rpm-based distro, happy to see Windows and macOS, we could improve our tests on those platforms, although I know we do not officially support them.

I'm confused... where is the mention of Windows and macOS here?  Or do you mean that they're available to GH Actions in general?

Reply:
> Or do you mean that they're available to GH Actions in general?

^this. They are available at GH Action.